utils/small_map/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "small_map"
```

Comment:
Both [`small_map`](https://crates.io/crates/small-map) and [`smallmap`](https://crates.io/crates/smallmap) are taken names. Maybe we should think of a different name? any ideas @boa-dev/maintainers ?

Reply:
`small_btree`?